marketplaceItems/soloPoint/appResources/appData/resources/js/soloPointTablet.js

```diff
@@ -0,0 +1,130 @@
+/*
+    Solo Point
+    Created by Milad Nazeri on 2019-01-07
+    Copyright 2016 High Fidelity, Inc.
+
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+    Point to solo someone to hear them better in a crowd!
+    TABLET JS
```

Comment:
That was just to signify it's the tablet ui so it would be clear, but the naming helps it.  I've always had a little bit on what the tablet JS should look like compared to the interface JS